models/calendar/dim_dates.sql

```diff
@@ -0,0 +1,136 @@
+/* gerando datas usando a macro do pacote dbt-utils */
+with
+    dates_raw as (
+    {{ dbt_utils.date_spine(
+        datepart="day",
+        start_date="cast('1970-01-01' as date)",
+        end_date="dateadd(year, 100, to_date(current_date()))"
+        )
+    }}
+)
+
+/* extraindo algumas informações da data e renomeando algumas colunas para o português */
+    , days_info as (
+        select
+            cast(date_day as date) as data_dia
+            , dayofweek(date_day) as dia_da_semana
+            , extract(month from date_day) as mes
+            , extract(quarter from date_day) as trimestre
+            , extract(dayofyear from date_day) as dia_do_ano
+            , extract(year from date_day) as ano
+            , to_char(date_day, 'DD-MM') AS dia_mes
+        from dates_raw
+    )
+
+/* renomeando os significados das colunas, traduzindo-os para o português */
+    , days_named as (
+        select
+            *
+            , case
+                when dia_da_semana = 1 then 'Segunda-feira'
```

Comment:
Is it possible to use these names in English? Just to try to make the package more "international"